docs/src/adding_models_for_general_use.md

```diff
@@ -800,7 +800,7 @@ target_scitype(SomeSupervisedModel) = Table(Continuous, Finite{2})
 restricts to tables with continuous or binary (ordered or unordered)
 columns.
 
-For predicting variable length sequences of, say, binary values
+For predicting variable-length sequences of, say, binary values
```

Comment:
Variable is an adjective (not a noun acting as an adjective) so I would beg to differ with Grammarly here. 

```suggestion
For predicting variable length sequences of, say, binary values
```

Reply:
Sure thing. I've double-checked that on Google and Wiki and that was also the more common use but happy to revert it.